src/layouts/SignedInAsWrapper.tsx

```diff
@@ -0,0 +1,37 @@
+import { Text } from "../components/atomic";
+import { useGetMyUserQuery } from "../generated/graphql";
+import { AuthorizationLevel, useAuthorizationLevel } from "../hooks";
+import { useAuth } from "../utils/firebase/auth";
+
+const SignedInAsWrapper: React.FC = ({ children }) => {
```

Comment:
This doesn't seem like it needs to be a wrapper--consider making it a component?